processor/src/errors.rs

```diff
@@ -93,7 +93,7 @@ pub enum ExecutionError {
         error: Box<dyn Error + Send + Sync + 'static>,
     },
     #[error("failed to execute Ext2Intt operation: {0}")]
-    Ext2InttError(Ext2InttError),
+    Ext2InttError(#[from] Ext2InttError),
```

Comment:
We had a discussion a while back about not using `#[from]` for readability purposes at the call site. I'm personally okay with either approaches though - but would be better if we were consistent.

cc @PhilippGackstatter